spec/clowne/integrations/active_record_dsl_spec.rb

```diff
@@ -0,0 +1,89 @@
+require "clowne/adapters/active_record/dsl"
+
+describe "AR DSl", :cleandb do
```

Comment:
Did you check it with a robocop?...double quotes ;)

Reply:
Yep) Fixed